.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        ruby: [ '2.6.3' ]
+      fail-fast: true
+    runs-on: ubuntu-latest
+    env:
+      CUSTOM_RUBY_VERSION: ${{ matrix.ruby }}
+    name: Ruby ${{ matrix.ruby }}
+    steps:
+      - name: Dump environment
+        run: env | sort
+      - name: Checkout ruby/www.ruby-lang.org
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      - name: Setup Ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          architecture: 'x64'
+      - name: Cache gem bundle
+        uses: actions/cache@v1
+        with:
+          path: vendor/bundle
+          key: bundle-${{ hashFiles('**/Gemfile.lock') }}
+      - name: Install bundler
+        run: gem install bundler
```

Comment:
would `--no-document` help here?

Reply:
Not much :smile:, but a good suggestion nevertheless.